lib/storage/file.js

```diff
@@ -321,6 +327,18 @@ nodeutil.inherits(File, ServiceObject);
  * });
  *
  * //-
+ * // If you pass in a string starting with "gs://" for the destination, the
+ * // file is copied to the other bucket and under the new name provided.
+ * //-
+ * file.copy('gs://other-bucket/my-image-copy.png', function(err, copiedFile, apiResponse) {
```

Comment:
It's okay to put the GS URL on another line:

```js
/* ...
 * // file is copied to the other bucket and under the new name provided.
 * //-
 * var newLocation = 'gs://another-bucket/my-image-copy.png';
 * file.copy(newLocation, function(err, copiedFile, apiResponse) {
 *   // `my-bucket` still contains:
 *   // - "my-image.png"
 *
 *   // `another-bucket` now contains:
 *   // - "my-image-copy.png"
 *
 *   // `copiedFile` is an instance of a File object that refers to your new
 *   // file.
 * });
```

^ Updated the bucket name and callback comments to match the next example as well